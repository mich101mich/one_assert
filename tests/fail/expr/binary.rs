fn main() {
    one_assert::assert!(1 + 1);
    one_assert::assert!(true == 1);
}
