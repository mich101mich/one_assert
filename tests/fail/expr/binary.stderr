error[E0308]: mismatched types
 --> tests/fail/expr/binary.rs:2:25
  |
2 |     one_assert::assert!(1 + 1);
  |                         ^^^^^ expected `bool`, found integer

error[E0308]: mismatched types
 --> tests/fail/expr/binary.rs:3:33
  |
3 |     one_assert::assert!(true == 1);
  |                         ----    ^ expected `bool`, found integer
  |                         |
  |                         expected because this is `bool`
